quay-enterprise/bitbucket-app.md

```diff
@@ -0,0 +1,19 @@
+# Creating an OAuth Application in BitBucket
+
+- Log into BitBucket
```

Comment:
Describe these steps really briefly?

"The first step is to set BitBucket to communicate with Quay:"